src/components/popover/popover.js

```diff
@@ -1,8 +1,12 @@
 import Vue from '../../utils/vue'
 import PopOver from '../../utils/popover.class'
 import warn from '../../utils/warn'
+import { getComponentConfig } from '../../utils/config'
 import normalizeSlotMixin from '../../mixins/normalize-slot'
 import toolpopMixin from '../../mixins/toolpop'
+import { HTMLElement } from '../../utils/safe-types'
```

Comment:
Could you move this up under `getComponentConfig`? we keep all the imports from the same directories together.